guide/creating-your-bot/event-handling.md

```diff
@@ -56,19 +70,33 @@ module.exports = {
 ```js
 module.exports = {
 	name: 'interactionCreate',
-	execute(interaction) {
-		console.log(`${interaction.user.tag} in #${interaction.channel.name} triggered an interaction.`);
+	async execute(interaction) {
```

Comment:
Not opposed, but something we'd want to be doing consistently everywhere in the guide. One thing I'm strongly trying to avoid is doing it one way then introducing a different method later.

Separate PR to migrate to that approach everywhere if enum usage is preferred.

Reply:
https://github.com/discordjs/guide/issues/1289